### DISCUSSION TITLE
file_download.py location

### DISCUSSION THREAD

**Discussion by Charmandrigo** [AbdBarho/stable-diffusion-webui-docker#533] [Q&A]
I am having a bug that freezes my computer entirely. it is fixed by editing a bad file
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/7901

I can't find this file, but the input of `DiffussionWrapper has 859.52 M params` appears meaning it's somewhere on it. There's not a venv folder so where is it?

**Comment by AbdBarho** [comment]
it is in the container, you might have to update the dockerfile manually to edit or replace the misbehaving file.
